defaults/main.yml

```diff
@@ -49,6 +54,23 @@ mon_check_pgsql: []
 # user: postgres
 # password: changeme
 
+mon_check_ldap: []
```

Comment:
A small line what exactly is checked would be nice in this instance - does it send only a bind request or does it do something else?